template/sample/template.gen.go

```diff
@@ -63,14 +63,22 @@ var (
 				if cpb.CheckExpression == "" {
 					return nil, fmt.Errorf("expression for field CheckExpression cannot be empty")
 				}
-
 				if t, e := tEvalFn(cpb.CheckExpression); e != nil || t != istio_mixer_v1_config_descriptor.STRING {
 					if e != nil {
 						return nil, fmt.Errorf("failed to evaluate expression for field CheckExpression: %v", e)
 					}
 					return nil, fmt.Errorf("error type checking for field CheckExpression: Evaluated expression type %v want %v", t, istio_mixer_v1_config_descriptor.STRING)
 				}
 
```

Comment:
Change in this File shows the support for map<string, int64|bool|double|string> types. PTAL